terraformer_iam.tf

```diff
@@ -0,0 +1,65 @@
+# Create the IAM instance profile for the Terraformer EC2 server
+# instances
+
+# The instance profile to be used
+resource "aws_iam_instance_profile" "terraformer" {
+  provider = aws.provisionassessment
+
+  name = "terraformer_instance_profile_${terraform.workspace}"
+  role = aws_iam_role.terraformer_instance_role.name
+}
+
+# The instance role
+resource "aws_iam_role" "terraformer_instance_role" {
+  provider = aws.provisionassessment
+
+  name               = "terraformer_instance_role_${terraform.workspace}"
+  assume_role_policy = data.aws_iam_policy_document.ec2_service_assume_role_doc.json
+}
+
+resource "aws_iam_role_policy" "terraformer_assume_delegated_role_policy" {
+  provider = aws.provisionassessment
+
+  name   = "assume_delegated_role_policy"
```

Comment:
Should this name include "terraformer" to be specific?

Reply:
Good call.  Please see commit 65663d2.